src/GameKit/GameKit.cs

```diff
@@ -144,6 +144,8 @@ public enum GKError : long {
 		FriendListDescriptionMissing = 100,
 		FriendListRestricted = 101,
 		FriendListDenied = 102,
+		[iOS (15,0), Mac (12,0), MacCatalyst (15,0), Watch (8,0), TV (15,0)]
```

Comment:
We usually don't add availability information on enum values that are errors, it doesn't really make much sense, since users aren't really using these values.

Reply:
Okay that makes sense! Thanks!